src/point_pillars.cpp

```diff
@@ -431,16 +431,19 @@ pybind11::array_t<float> createPillarsTarget(
     // zone-in on potential spatial area of interest
     float objectDiameter =
         std::sqrt(std::pow(labelBox.width, 2) + std::pow(labelBox.length, 2));
-    const auto offset = static_cast<int>(
+    const auto xOffset = static_cast<int>(
```

Comment:
@RaphvK Do you know more about the offset modification from Max? From my point of view, the original implementation (seperation in x and y makes more sense). What you you think?

Reply:
No idea why he changed that, maybe @bastilam has an idea?

Reply:
Unfortunately, I do not.